ui/src/app/pages/analysis/cohort-resource-card.tsx

```diff
@@ -0,0 +1,182 @@
+import {RenameModal} from 'app/components/rename-modal';
```

Comment:
This should not be under `/analysis`. It should probably be under `/data/cohorts`. (If I remember the path correctly)